### PR TITLE
Fortmatic integration via ethereum.sendAsync payload id and jsonrpc key

### PR DESCRIPTION
**Pull request by rej156**
BOOM offchain profiles with personal_sign and no metamask!

### CODE REVIEW

File: src/utils/index.js [hunk 1]
```diff
@@ -28,6 +30,8 @@ module.exports = {
     var method = 'personal_sign'
     return new Promise((resolve, reject) => {
       ethereum.sendAsync({
+	jsonrpc: "2.0",
```
Comment (oed) [inline src/utils/index.js]: Missing a white space here 🙂

File: src/utils/index.js [hunk 2]
```diff
@@ -49,6 +53,8 @@ module.exports = {
...
```
Comment (oed) [inline src/utils/index.js]: Same as above

**Comment by rej156** [comment]
https://github.com/ethereum/web3.js/issues/1967

No idea really what the payload id is if I'm honest, seems internal and is reported here
Feels like an implementation detail for concurrency I guess

jsonrpc is very standard yup

Will amend this PR!
I also think we need to change the fromAddress key to from for standardisation too infact

**Comment by oed** [comment]
So does `id` need to be the date, or can it just be `0` or whatever?

Do you know where the definition of `personal_sign` resides online?

**Comment by rej156** [comment]
Looks like a web3.js niggle.
https://github.com/ethereum/go-ethereum/wiki/Management-APIs#personal_sign
In our kauri repo, we set it as Date but yeah it can be 0 or whatever

**Comment by oed** [comment]
Ok, I think I would prefer just `0` then, unless there is a good reason to use Date.

Strange, so in that documentation it says `account`, `fromAddress` is currently being used, and you suggested `from`. I think I'd rather just stay with what we have right now until it becomes a problem 🙂

**Comment by rej156** [comment]
Np makes sense! Will amend based on what you said after this coffee

**Comment by oed** [comment]
Thanks 😊

**Comment by rej156** [comment]
@oed done, sorry not sorry about the prettier plugin on my editor 🥇

**Comment by oed** [comment]
Great, thanks @rej156 !

**Pull request merged**